en/ros2/px4_ros2_interface_lib.md

```diff
@@ -1,5 +1,17 @@
 # PX4 ROS 2 Interface Library
 
+:::warning Experimental
+At time of writing parts of the PX4 ROS 2 Interface Library is experimental, and hence subject to change:
+
+- The architecture and core interfaces for defining modes in ROS 2 modes are largely stable.
+  The library offers significant benefits over using offboard mode in its current state.
+- Only a few setpoint types have settled (the others are still under development).
+  You may need to use internal PX4 topics which may not remain backwards-compatible over time.
+- The API is not fully documented.
+- Testing in CI is still limited.
```

Comment:
This is ok, but just fyi most of the things that are there are tested. Mostly the setpoint types (with their expected behavior) that are not.

Reply:
Then let's say that :-)